setup.py

```diff
@@ -14,16 +14,15 @@
     long_description=open('README.rst', 'r').read(),
     packages=find_packages(),
     install_requires=[
-        'django>=1.8,<1.9a1',
-        'djangorestframework>=3.1',
+        'django>=1.9,<1.10a1',
+        'djangorestframework>=3.3',
```

Comment:
Do we really need the 3.3 version or is this nice to have? The idea is that requirements in setup.py are as loose as possible so that other people can include our app into their Django projects even if they use older versions required modules. The same goes for django-filter and jsonschema. Of course it makes sense to raise the requirements if there were security fixes or added features that we need.